.travis.yml

```diff
@@ -1,10 +1,43 @@
 # See Dockerfile for the full build instructions
 sudo: required
-language: cpp
+language: python
+cache: pip
+
+#mainly for flake8 as everything else runs in docker
+python: 
+  - 3.5
+
+git:
+  quiet: true
 
 services:
   - docker
 
-script:
-  - docker build .
-
+jobs:
+  include:
+    - stage: PythonTests
```

Comment:
Also the "SimTests" are using Python, making the name "PythonTests" a bit ambiguous in my eyes. How about "lint" or "pythonlint"?